labs/src/main/scala/org/scalalabs/basic/lab03/RecursionPatternMatchingExercise.scala

```diff
@@ -69,7 +82,7 @@ object RecursionPatternMatchingExercise {
    * List(List(1,2,3), List('A, 'B, 'C), List('a, 'b, 'c)) -> List(List(1, 'A, 'a), List(2, 'B, 'b), List(3, 'C, 'c))
    */
   def zipMultiple(in: List[List[_]]): List[List[_]] = {
-    error("fix me")
+    in.transpose
```

Comment:
Oo. Very nice. This reminds me of Python, where `zip(*L)` literally means zipMultiple, but is often used for transpose.

I think this is clever enough to allow for the fact that it isn't a recursive solution.